products/satellite.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Red Hat Satellite Server
+category: server-app
+changelogTemplate: "https://access.redhat.com/documentation/en-us/red_hat_satellite/__RELEASE_CYCLE__/html/release_notes/index"
+releases:
+-   releaseCycle: "6.12"
+    eol: 2024-05-31
+    support: 2023-05-31
+    releaseDate: 2022-11-16
+    latest: "6.12.0"
+    latestReleaseDate: 2022-11-16
+-   releaseCycle: "6.11"
+    eol: 2024-01-31
+    support: 2022-11-30
+    releaseDate: 2022-07-05
+    latest: "6.11.4.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.10"
+    eol: 2023-05-31
+    support: 2022-06-30
+    releaseDate: 2021-11-16
+    latest: "6.10.7.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.9"
+    eol: 2022-11-30
+    support: 2021-11-30
+    releaseDate: 2021-11-16
+    latest: "6.9.10"
+    latestReleaseDate: 2022-11-17
+iconSlug: redhat
+permalink: /satellite
+alternate_urls:
+-   /rhsat
+releasePolicyLink: https://access.redhat.com/support/policy/updates/satellite
+activeSupportColumn: Full support
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Maintenance support
+discontinuedColumn: false
+versionCommand: yum info satellite
+
+---
+
+[Red Hat® Satellite](https://www.redhat.com/technologies/management/satellite) is an infrastructure management product specifically designed to keep 
+Red Hat Enterprise Linux® environments and other Red Hat infrastructure running efficiently, with security, and compliant with various standards.
+
+There is a new minor release approximately every 6 months.
```

Comment:
Maybe something like :

> Red Hat delivers a a new minor release approximately every 6 months. Each release have full support for six month, then enters in maintenance support for one year. Additional release infos can be found on [https://access.redhat.com/articles/1365633#sat6](https://access.redhat.com/articles/1365633#sat6).

The first link is also the `releasePolicyLink`, so I do not think it is necessary to keep it.